gui/GiveItem.java

```diff
@@ -82,19 +92,9 @@ private void giveItemToPlayer() {
 		if (itmSt != null) {
 			itemMod = " " + itmSt.getItemDamage();
 		}
-		String amt;
-		if (!((amountBox.getText() == null) || amountBox.getText().isEmpty())) {
-			try {
-				int amount = Math.abs(Integer.parseInt(amountBox.getText())) % 65;
-				amt = "" + amount;
-			} catch (NumberFormatException nfe) {
-				amt = "1";
-			}
-		} else {
-			amt = "1";
-		}
-		Minecraft.getMinecraft().thePlayer.sendChatMessage(
-				"/give " + userBox.getText() + " " + tItem.getRegistryName() + " " + amt + " " + itemMod);
+		String amt = amountBox.getText() == null ? "1" : amountBox.getText();
+		Minecraft.getMinecraft().thePlayer.sendChatMessage("/give " + userBox.getText().split("-")[0] + " "
+				+ GameRegistry.findUniqueIdentifierFor(tItem) + " " + amt + " " + itemMod);
```

Comment:
I think this needs to change back to the code on the left... I think using the game registry said to use the method
tItem.getRegistryName() instead of the find unique identifier